static/app/views/settings/account/notifications/utils.tsx

```diff
@@ -40,9 +40,9 @@ export function getDocsLinkForEventType(
 ) {
   switch (event) {
     case 'transaction':
-      return 'https://docs.sentry.io/product/performance/transaction-summary/#what-is-a-transaction';
+      return 'https://sentry-docs-git-move-quota-to-billing.sentry.dev/pricing/quotas/manage-transaction-quota/';
```

Comment:
why is this link so strange? I tested it and it works but still

Reply:
weird cuz then going anywhere else on docs also uses that domain

Reply:
this is the domain we're putting am2 docs on for now

Reply:
ah ok makes sense

Reply:
i'll double check with the team but this is specifically for billing with transactions (old) vs billing with spans (new)